app/Services/MovieServiceImpl.php

```diff
@@ -46,18 +46,32 @@ public function findOne($id){
     public function search($searchParam){
         return  Movie::where('title', $searchParam)->with('reactions')->get();
     }
+    public function elasticSearch($searchParam){
+        $movies= Movie::complexSearch(array(
+            'body' => array(
+                'query' => array(
+                    'match' => array(
+                        'title' => $searchParam,
+                    )   
+                )
+            )
+        ));
+        return $movies;
```

Comment:
jel ti ovo vraca cele modele?

Reply:
trebalo bi da vraca i lajkove, sad sam namestio da indeksira ceo model

Reply:
bolja opcija je da se indeksiraju polja koja pretrazujes (title, description...) i id, i kada se izvrsi query u elasticsearchu da dobijes id-eve koji ti trebaju i onda to cupas iz baze, ali ne moras to sad raditi